server/service/src/sync/translation_and_integration.rs

```diff
@@ -72,12 +66,8 @@ impl<'a> TranslationAndIntegration<'a> {
                 Err(translation_error) => {
                     self.sync_buffer
                         .record_integration_error(&sync_record, &translation_error)?;
-                    result.insert_error(
-                        &sync_record.table_name,
-                        Some(&sync_record),
-                        None,
-                        translation_error,
-                    );
+                    result.insert_error(&sync_record.table_name);
+                    warn!("{:?}", translation_error);
```

Comment:
reinstate log here, simpler then TranslationAndIntegration result with errors (originally needed for pull and push test where errors were hidden, but just added log initialisation to that test